### PR TITLE
PADV-283.1 - Change support text style

### PR DESCRIPTION
**Pull request by sergivalero20**
## Description

This PR changes support text style in basket page.

## Ticket for this PR:

[PADV-283](https://agile-jira.pearson.com/secure/RapidBoard.jspa?rapidView=51807&projectKey=PADV&view=detail&selectedIssue=PADV-283).

## Worked on this PR

- [x] Align support text in basket template.

## Screenshots

image

### CODE REVIEW

File: ecommerce/certprep/templates/oscar/basket/partials/hosted_checkout_basket.html [hunk 1]
```diff
@@ -61,17 +61,18 @@
...
-                                        {% trans "Pearson VUE Support" %}
-                                    </a>.
-                                </p>
                             {% endif %}
                         </div>
                     </div>
+                    {% if not free_basket %}
+                        {% settings_value 'payment_support_email' as support_email %}
+                        {% settings_value 'payment_support_url' as support_url %}
+                        <p id="support-text">{% trans "The enrollment link is invalid, please contact" %}
```
Comment (alexjmpb) [inline ecommerce/certprep/templates/oscar/basket/partials/hosted_checkout_basket.html]:
```suggestion
                        <p class="support-text">{% trans "The enrollment link is invalid, please contact" %}
```

What do you think about using a class name instead of an ID to style?

Comment (sergivalero20) [reply ecommerce/certprep/templates/oscar/basket/partials/hosted_checkout_basket.html]: Well, I'm using that element only once, so I figured it would be okay to use an id instead of a class name. But, I can change it if it's necessary.

**Review by alexjmpb — APPROVED** [review]

File: ecommerce/certprep/templates/oscar/basket/partials/hosted_checkout_basket.html [hunk 2]
```diff
@@ -61,17 +61,18 @@
...
-                                {% settings_value 'payment_support_url' as support_url %}
-                                <p>{% trans "The enrollment link is invalid, please contact" %}
-                                    <a {% if support_email %} href=mailto:{{ support_email }} {% else %} href={{ support_url }} {% endif %}>
-                                        {% trans "Pearson VUE Support" %}
-                                    </a>.
-                                </p>
                             {% endif %}
                         </div>
                     </div>
+                    {% if not free_basket %}
```
Comment (Squirrel18) [inline ecommerce/certprep/templates/oscar/basket/partials/hosted_checkout_basket.html]: Why did you add this validation again? @sergivalero20

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: ecommerce/certprep/templates/oscar/basket/partials/hosted_checkout_basket.html (hunk 2 above)
Comment (sergivalero20) [reply ecommerce/certprep/templates/oscar/basket/partials/hosted_checkout_basket.html]: Sorry, I already removed it.

**Review by Squirrel18 — APPROVED** [review]

File: ecommerce/certprep/static/sass/base/_basket.scss [hunk 3]
```diff
@@ -174,6 +176,14 @@ $basket-items-color: #ddd;
                     background: darken($brand-primary, 10%);
                 }
             }
+
+            .support-text {
+                position: absolute;
```
Comment (Squirrel18) [inline ecommerce/certprep/static/sass/base/_basket.scss:181]: Did you test this in mobile views? @sergivalero20

Comment (sergivalero20) [reply ecommerce/certprep/static/sass/base/_basket.scss:181]: Yes, I did @Squirrel18 

image

**Review by alexjmpb — APPROVED** [review]

**Review by Squirrel18 — APPROVED** [review]
